web/clusters.go

```diff
@@ -549,12 +502,30 @@ func NewClusterHandler(client consul.Client) gin.HandlerFunc {
 		}
 
 		c.HTML(http.StatusOK, "cluster_hana.html.tmpl", gin.H{
-			"Cluster":          cluster,
+			"Cluster":          clusterItem,
 			"Nodes":            nodes,
-			"StoppedResources": stoppedResources(cluster),
+			"StoppedResources": stoppedResources(clusterItem),
 			"ClusterType":      clusterType,
 			"HealthContainer":  hContainer,
-			"ChecksCatalog":    checks,
+			"ChecksCatalog":    checksCatalog,
 		})
 	}
 }
+
+type checkSelectionForm struct {
+	Ids []string `form:"ids[]"`
+}
+
+func NewPostClusterHandler(client consul.Client) gin.HandlerFunc {
+	return func(c *gin.Context) {
```

Comment:
Maybe i would call it SaveChecks handler

Reply:
Ok, I like it